packages/compas-convert/src/passes/convert-test-files.ts

```diff
@@ -187,29 +184,13 @@ function handleNestedTest(expression: CallExpression) {
 			}
 		}
 	}
-
-	if (
-		usage.find((it) =>
-			[
-				TestCommand.equal,
-				TestCommand.deepEqual,
-				TestCommand.notEqual,
-				TestCommand.ok,
-				TestCommand.notOk,
-				TestCommand.pass,
-				TestCommand.fail,
-			].includes(it.command),
-		)
-	) {
-		addNamedImportIfNotExists(expression.getSourceFile(), "vitest", "expect", false);
-	}
 }
 
 /**
  * Check if the expression only contains other tests
  */
 function testIsParent(expression: CallExpression): boolean {
-	return getDirectDescendants(expression).every((child) =>
+	return getDirectDescendants(expression).some((child) =>
```

Comment:
A `t.test()` should also become a `describe` when it has call expressions that aren't necessarily child `t.test`